tests/conftest.py

```diff
@@ -29,3 +30,8 @@ def executor(connection):
 @pytest.fixture
 def exception_formatter():
     return lambda e: str(e)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def temp_config(tmpdir_factory):
```

Comment:
Where is this fixture being used?

Reply:
It is used automatically on session start